Sourcery/Models/Type.swift

```diff
@@ -169,7 +169,6 @@ public class Type: NSObject, SourceryModel, Annotated {
     }
 
     // sourcery: skipEquality, skipDescription
-    /// Contained types groupd by their names
     public private(set) var containedType: [String: Type] = [:]
```

Comment:
Except that it supposed to post warning only. But I guess we can just fail always 😈